src/pages/index.js

```diff
@@ -183,18 +183,6 @@ document.addEventListener("DOMContentLoaded", () => {
     }
   }
 
-  constants.profileEditButton.addEventListener("click", () => {
-    profileEditValidator.resetValidation();
-    const userData = userInformation.getUserInfo();
-    if (userData) {
-      constants.profileTitleInput.value = userData.name || "";
-      constants.profileDescriptionInput.value = userData.about || "";
-      profileEditForm.open();
-    } else {
-      console.error("User data is undefined");
```

Comment:
You've deleted the right profile form opening. You need to delete the code where you call `api.getUserInfo()` on opening which is not needed because you have method `userInformation.getUserInfo();` for that